server/app/featureflags/FeatureFlag.java

```diff
@@ -33,7 +33,10 @@ public enum FeatureFlag {
   NONGATED_ELIGIBILITY_ENABLED,
 
   // Phone number question type.
-  PHONE_QUESTION_TYPE_ENABLED;
+  PHONE_QUESTION_TYPE_ENABLED,
+
+  // New login form
+  NEW_LOGIN_FORM_ENABLED;
```

Comment:
should we add this flag to the featureflag.conf file?

Reply:
Yes - thanks for the reminder.